### ISSUE TITLE
Page now outdated

### ISSUE THREAD

**Issue by aldoug** [MicrosoftDocs/microsoft-365-docs#736]
I am following up on updating this, as it appears that only some safety tips are still supported

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d1daade0-c937-6daf-cdc5-e6e23214426d
* Version Independent ID: 49836613-b9a3-fc37-ffc8-3589dda16d45
* Content: [Safety tips in email messages in Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/safety-tips-in-office-365?source=docs)
* Content Source: [microsoft-365/security/office-365-security/safety-tips-in-office-365.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/safety-tips-in-office-365.md)
* Service: **o365-seccomp**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by mypil** [comment]
@aldoug  - Thank you for submitting feedback.

I will get this issue over to the Microsoft 365 writing team for investigation.

Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.

**Comment by aldoug** [comment]
Thanks Joyce

If I can be of any assistance in the amendment process please feel free to reach out via my alias “aldoug”

Alisdair Douglas
Support Escal Eng
Eq-Uk Gtsc Bizapps Server Customer Services and Support
Office: +44 (118) 9095190
aldoug@microsoft.com<mailto:aldoug@microsoft.com>
Working hours: Monday – Friday 08:00 - 16:30
[Microsoft Logo]
Backup | Jeroen Dronkers (jedronke@microsoft.com<mailto:jedronke@microsoft.com>)
Team Manager | Frances Early (francese@microsoft.com<mailto:francese@microsoft.com>) +44 (20) 34251539

Did you know? You can submit your feedback for this Support ticket from the Navigation menu in the Microsoft 365 admin center via ‘Support’ (headset icon)/‘View service requests’/‘Ticket #’ (see subject of this email).


This and subsequent emails on the same topic are for discussion and information purposes only. Only those matters set out in a fully executed agreement are legally binding. This email may contain confidential information and should not be shared with any third party without the prior written agreement of Microsoft. If you are not the intended recipient, take no action and contact the sender immediately.

Microsoft Limited (company number 01624297) is a company registered in England and Wales whose registered office is at Microsoft Campus, Thames Valley Park, Reading RG6 1WG

**Comment by mypil** [comment]
@aldoug - We value collaboration and appreciate your efforts to clarify the docs.

Could you please assist us by providing more details on how this content could be edited to incorporate your feedback? If you could suggest what the appropriate text and where it should be placed, I could assign one of our team members to create a Pull Request to get it added or you may also create a Pull Request to add in your suggestion and just link this issue so I can have the doc owner review and merge the content. Thank you.

**Comment by aldoug** [comment]
Sure

For the https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/safety-tips-in-office-365 page, I am still following up to find out our expectations. I am reaching out to Ross Adams on this one as apparently it is in the backlog

Alisdair Douglas
Support Escal Eng
Eq-Uk Gtsc Bizapps Server Customer Services and Support
Office: +44 (118) 9095190
aldoug@microsoft.com<mailto:aldoug@microsoft.com>
Working hours: Monday – Friday 08:00 - 16:30
[Microsoft Logo]
Backup | Jeroen Dronkers (jedronke@microsoft.com<mailto:jedronke@microsoft.com>)
Team Manager | Frances Early (francese@microsoft.com<mailto:francese@microsoft.com>) +44 (20) 34251539

Did you know? You can submit your feedback for this Support ticket from the Navigation menu in the Microsoft 365 admin center via ‘Support’ (headset icon)/‘View service requests’/‘Ticket #’ (see subject of this email).


This and subsequent emails on the same topic are for discussion and information purposes only. Only those matters set out in a fully executed agreement are legally binding. This email may contain confidential information and should not be shared with any third party without the prior written agreement of Microsoft. If you are not the intended recipient, take no action and contact the sender immediately.

Microsoft Limited (company number 01624297) is a company registered in England and Wales whose registered office is at Microsoft Campus, Thames Valley Park, Reading RG6 1WG

**Comment by scottsa-msft** [comment]
My customer is also complaining that the data on this site is out of date.   The SPAM links are listed as yellow but are now showing as Red.   Also they see new tips.   Are those Safety Tips or other tips?   Have we added new tips now that are not listed on this site?    Please update the site to show the current functionality.

**Comment by mypil** [comment]
@aldoug - Do you have any updates on this?

Thank you.

**Comment by DeepRaj98** [comment]
Could anyone confirm when can we expect updated details on this page? There is a slight difference in the format of these safety tips and these are not visible in all expected scenarios.  I had opened a case with MS (16920738) and I got below feedback last month but not sure about the ETA - "We tested the behavior and found that the issue of safety tip is by design. Only user impersonated emails will show consistent safety tip in Outlook and OWA, including emails in Junk folder. We are in the process of documenting this."
